### ISSUE TITLE
App available outside Play Store?

### ISSUE THREAD

**Issue by IzzySoft** [amnesica/ClearClipboard#1]
Would you consider [tagging](https://github.com/amnesica/ClearClipboard/releases) your releases and attaching the APK files to their corresponding tag, so your app becomes available to those outside the walled garden of Google Play? Not everyone has access there, for different reasons (moslty because their devices have no GApps – either willingly, for privacy reasons, or because the manufacturers didn't include them). Thanks in advance!

**Comment by amnesica** [comment]
Hi @IzzySoft, thanks for the suggestion, you can now see the current release in the release section

**Comment by IzzySoft** [comment]
Thanks – that was fast! For the convenience of the said user group, your app will [become available here](https://apt.izzysoft.de/fdroid/index/apk/com.amnesica.clearclipboard) at around 19:00 UTC today. Most of those users are using one of the available [F-Droid clients](https://android.izzysoft.de/applists/category/named/apps_markets#group_1181) instead of the Play Store app, and many having my repo configured there – so your app will be available to them easily, and updates are notified to them automatically (my updater checks all 24h and grabs new APKs).

If you like what you will see there after it became available, be welcome to [add the corresponding badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) next to the one of Play Store :smiley: And if you wish to proceed to [F-Droid](https://f-droid.org/) itself, which your app seems eligible for (I see no stoppers so far), I gladly help you with that.

PS: Your APK is compiled for debug. Not a big issue, but I thought to let you know in case that was not intended.

**Comment by amnesica** [comment]
@IzzySoft Thanks for the interest in the app and the hint with the debug APK, I've just replaced the debug APK with a release APK :-)

I would like to take your help to get this app and [Feedsta](https://github.com/amnesica/Feedsta), if it seems eligible, into the F-Droid store. What steps are necessary to do this?

**Comment by IzzySoft** [comment]
Thanks for picking up the offer! If this is your first time with bringing an app to F-Droid, the easiest approach is [opening a Request For Packaging](https://gitlab.com/fdroid/rfp/-/issues), and we guide you through. No technical knowledge needed for that – and I expect it to go like a breeze. Once all looks good at RFP, the next step would be to proceed to a Merge Request at fdroiddata, to get the metadata set up from which your app will be built. Again, no rocket-science – but that step would need some technical knowledge. But the team is there to help – and if you feel unable to do the MR yourself, we'll do it for you.

Let me have one more look at the repo here; if I remember correctly… yupp, I do: there're no [Fastlane structures](https://gitlab.com/-/snippets/1895688). I can offer to send you what I set up with my repo, by opening a PR here with a starter package. We can straighten that out together then, so you can set it up for Feedsta yourself (I prefer not to be involved with any zucking stuff, apologies ahead). As a TL;DR background: it's about summary, description and screenshots to be placed in your repo in a common structure, so you have full control on how your app is presented while we¹ can pull and integrate that automatically.

¹ disclosure: I'm one of the F-Droid maintainers. And hundreds of apps have walked through my repo to finally find their way into F-Droid :smiley:

**Comment by amnesica** [comment]
Hi @IzzySoft sorry for coming back at this one year later, but since I just made a merge request for KryptEY I thought about adding ClearClipboard to F-Droid as well. Since you're already hosting it in your repo, do you think it makes sense to publish it in F-Droid?

**Comment by IzzySoft** [comment]
It sure does! F-Droid.org has a higher trust level than my repo. Ideally you achieve "reproducible builds" – which means F-Droid builds from your source, compares the resulting APKs with those you provide and, if that succeeds, ships yours. That way it becomes clear your APK is really what it claims to be, and nobody tampered it. This won't be able with your listing in my repo – which you then still could uphold for e.g. faster updates (as your APKs would be shipped on both repos, cross-updates would be possible).

TL;DR: feel encouraged to go for a listing at F-Droid.org, please! It's a win for you and for all people using your app.

**Comment by amnesica** [comment]
@IzzySoft thanks for the fast reply :) Will prepare the f-droid release

**Comment by IzzySoft** [comment]
@amnesica Congrats: your app is now available at F-Droid! So maybe you want to replace the badge in your readme? Or do you want me to keep the app in my repo as well, and thus just add the F-Droid badge then?

**Comment by IzzySoft** [comment]
@amnesica ^^ ping :smiley:

**Comment by amnesica** [comment]
Hi @IzzySoft, you can keep the app in your repo and I will add the F-Droid badge here. Just did not have the time yet :)

**Comment by IzzySoft** [comment]
Thanks, marked for "keep" then :smiley:
